packages/field-label/src/FieldLabel.ts

```diff
@@ -77,6 +75,9 @@ export class FieldLabel extends SpectrumElement {
         }
         const parent = this.getRootNode() as HTMLElement;
         const target = parent.querySelector(`#${this.for}`) as Focusable;
+        if (!target) {
+            return;
+        }
```

Comment:
Can you add a unittest that checks this as well?

Reply:
I was trying to make that work, but was having trouble getting that error to actually catch anywhere beyond it printing to the console. I will take another pass, but if you have any experience to share around testing that errors are/are not thrown, I'd very much appreciate it!

Reply:
Since this is an async function the throw will convert to a rejected promise. That might be why you're not finding it with exceptions? I've made some progress by changing `manageFor` to be public so I can spy on it. Didn't quite crack it though -- here's where I stopped:

```
    it('loads with invalid "for"', async () => {
        const el = await fixture<FieldLabel>(
            html`
                <sp-field-label for="not_exist">Input label</sp-field-label>
            `
        );
        const manageSpy = sinon.spy(el, 'manageFor')
        await elementUpdated(el);

        const result = manageSpy.returnValues[0]; // result is null here, and not a promise :(
        try {
            await result;
        } catch {
            expect(true).to.be.false;
        }
        await expect(el).to.be.accessible();
    });
```

Reply:
I'm also changing this PR to approved. It does seem like we'll want to figure out a way to do a little deeper testing/validation (but maybe in some of these cases we just need to add some error state we can check when things aren't right?)

Reply:
or maybe I'll check in a really basic test so that you'd at least see the console errors in the test output pre-fix.

Reply:
This got me doing in the right direction, @adixon-adobe! I've got a test outlining the ability to have `for` that doesn't point to anything without throwing an error.

This does point to needing to add a MutationObserver later to support late additions of `for` value resolutions, but let's address that it a separate PR.